### PR TITLE
Link from 'configuration' to 'introduction'

### PR DESCRIPTION
**Pull request by raboof**
And add a link from Configuration to Introduction

### CODE REVIEW

File: docs/src/main/paradox/configuration.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Configuration
 
-Just like any other Akka module Akka HTTP is configured via [Typesafe Config](https://github.com/lightbend/config).
+Before configuring Akka HTTP it must be added to your project as described in
+@ref[the introduction](introduction.md#using-akka-http)
+
+Just like any other Akka module Akka HTTP is configured via [Lightbend Config](https://github.com/lightbend/config).
```
Comment (johanandren) [inline docs/src/main/paradox/configuration.md]: Isn't the library still called "typesafe config" though?

Comment (raboof) [reply docs/src/main/paradox/configuration.md]: Oops! Not sure what I was thinking.

File: docs/src/main/paradox/configuration.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Configuration
 
+Before configuring Akka HTTP it must be added to your project as described in
+@ref[the introduction](introduction.md#using-akka-http)
```
Comment (jrudolph) [inline docs/src/main/paradox/configuration.md]:
```suggestion
@ref[the introduction](introduction.md#using-akka-http).
```

**Comment by raboof** [comment]
The rationale is that I wanted to point a user to the docs on adding the dependency, and it took me a few attempts to find it myself - so I wanted to add some more pointers in the right direction. Indeed better headings/subpages might help even more.

**Comment by jrudolph** [comment]
I see, thanks, @raboof.

**Pull request merged**